src/@next/Collapse/CollapseItem.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+
+import { Icon } from '../Icon';
+import { Typography } from '../Typography';
+import { Neutral } from '../utilities/colors';
+import {
+  CollapseContent,
+  CollapseContentWrapper,
+  CollapseItemHeader,
+  CollapseItemContent,
+  IndicatorIconWrapper,
+  RightHeaderSection,
+} from './CollapseItemStyle';
+
+export interface CollapseItemProps {
+  /** Content to be displayed when the item is not collapsed, 8px padding applied by default */
+  children: React.ReactNode;
+  /** Title for each collapse item on the header, simply can pass in a string */
+  title: React.ReactNode;
+  /** Extra components to be displayed on the left side of heading, on the right of indicator (if it is left-aligned) and header (title), flex row with gap of 4px is applied */
+  headerLeft?: React.ReactNode;
+  /** Extra components to be displayed on the right side of heading, on the left of indicator (if it is right-aligned), flex row with gap of 4px is applied */
+  headerRight?: React.ReactNode;
```

Comment:
yea but it will be harder for people to add items on left or right;
if there is no `headerLeft` and `headerRight`, you need to manually create the row flex container, with 2 divs inside (one for items at left, and the other for right), since the sample in Figma show some badges & icons used on left & right, I made it like this so that it is easier;
also, I put the `title` inside Typography directly with the appropriate color (depending on disabled or not)

please reply again if you still want me to remove `headerLeft` and `headerRight`